cura/CuraPackageManager.py

```diff
@@ -55,6 +57,24 @@ def initialize(self) -> None:
 
         super().initialize()
 
+    def isMaterialBundled(self, file_name: str, guid: str):
+        """ Check if there is a bundled material name with file_name and guid """
```

Comment:
I think you should also write down the documentation that you put into the PR here as well. The logic that a "secureSearchPath" is bundled is info that you would want here.